### DISCUSSION TITLE
Python Programlamaya Giriş 2 – Sayılar, Aritmetik, Temel Veri Yapıları | SonsuzUs Github

### DISCUSSION THREAD

**Discussion by giscus** [sonsuzus/sonsuzus.github.io#130] [Site]
# Python Programlamaya Giriş 2 – Sayılar, Aritmetik, Temel Veri Yapıları | SonsuzUs Github

Yeni başlayanlar için Python programlamayı anlatan yazı dizimizin ikinci bölümünde sayıları, aritmetik işlemlerini, ve Python’daki temel veri tiplerini (listeler, çokuzlar, dizeler, sözlükler) işliyoruz.

https://sonsuzus.github.io/posts/python-programlamaya-giris-sayilar-aritmetik-temel-veri-yapilari/

**Comment by sonsuzus** [comment]
İlginç bir konu.
